packages/dd-trace/src/plugins/outbound.js

```diff
@@ -25,6 +26,14 @@ class OutboundPlugin extends TracingPlugin {
     })
   }
 
+  startSpan (...args) {
+    const span = super.startSpan(...args)
+    if (this._tracerConfig.codeOriginForSpans.enabled) {
+      span.addTags(exitTag(this.startSpan))
```

Comment:
nit: would suggest rename entryTag and exitTag method names. when reading the code it look like it add just one tag, which is misleading. I was thinking... createCodeOriginTagsForExitSpan(this.startSpan) but that might be a mouthful

Reply:
I agree they could probably be renamed from singular to plural 👍

But I think the method names should make sense locally to the module they live in. In this case, they live inside the `datadog-code-origin` module, so adding something like "code origin" to their name is repeating information that's already present in the module name. Similar to how the functions inside of the `path` module doesn't repeat the word `path` (e.g. it's just `join` instead of `pathJoin`).

If one wants to make them distinct in the importing module, one would either import the entire module:

```js
const codeOrigin = require('../../../datadog-code-origin')
// ...
codeOrigin.exitTags()
```

Rename them on import:

```js
const { exitTags: codeOriginExitTags } = require('../../../datadog-code-origin')
// ...
codeOriginExitTags()
```

Or you just let the reader look up where they are imported from - like today.

My general rule, unless there's a critical risk of misunderstanding what's going on, is to just leave their names as is, even though it's tiny bit harder to parse the code. I feel this is the normal way naming is done in most Node.js code.

Reply:
I've updated the code now with the plural method names